Dockerfile

```diff
@@ -104,4 +108,5 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vendor="Torquem" \
       org.label-schema.version=$VERSION
 
-ENTRYPOINT ["erigon"]
+# Entrypoint is messing up with compose
```

Comment:
I'd rather override entry point in `x-erigon-service` in docker-compose to have as few changes in this file as possible

Reply:
Will try to do that